.env.test

```diff
@@ -88,6 +88,7 @@ OAUTH2_CLIENT_ID=VECGuQb00tWt8HZNkA4cxu6dnoQD5pF6Up3daAoK
 OAUTH2_CLIENT_SECRET=aY14rwkEKasNqBEZX8OnhpRk8lpHAfT7oKTlf4LriEK8oMZxhnGKcnt4bZ72pceNEl83B6LtBvhKr3BqBLFA80Pd6Ugav2rkc8bk7TE4LkaoB2qcBQmjiOiEpizsgZGx
 OAUTH2_CLIENT1_ID=test-id-for-client-with-pkce-flow
 OAUTH2_AUTH_URL=http://localhost:8083/o/authorize
+OAUTH2_USER_PROFILE_URL=http://layman_test_run_1:8030/rest/test-oauth2/user-profile
```

Comment:
Try to unset this env. (Problem is in test `src/layman/authn/oauth2_test.py::test_token_active`)